### DISCUSSION TITLE
Parcel 2 - Protect Folder Structure

### DISCUSSION THREAD

**Discussion by hoemon** [parcel-bundler/parcel#4629] [General] [answered]
I've just installed yarn to test out parcel 2 as 1 didn't support keeping ther existing folder structure. Now I'm wondering, what's the command line argument for keeping the folder structure as it is?

**Comment by DeMoorJasper** [comment]
If you mean maintain the source file structure in the dist folder it is currently not the default behaviour and would require a custom namer plugin to do

**Comment by mischnic** [comment, marked as answer]
@hobofan I've created an experimental namer to do just that (it's really just the default namer with a commented out if statement): https://www.npmjs.com/package/@mischnic/parcel-namer-preserve-structure

**Reply by hobofan** [reply]
I'm assuming you meant to ping @hoemon ? 😅

**Reply by mischnic** [reply]
I'm sorry 😄

**Reply by hoemon** [reply]
Is this v2 only?

**Reply by mischnic** [reply]
Yes

**Reply by hoemon** [reply]
:(
